app/routes.tsx

```diff
@@ -485,6 +485,18 @@ export const routes = createRoutesFromElements(
             path="access"
             lazy={() => import('./pages/project/access/ProjectAccessPage').then(convert)}
           />
+          <Route
+            path="affinity"
+            lazy={() => import('./pages/project/affinity/AffinityPage.tsx').then(convert)}
+          />
+          <Route path="affinity" handle={{ crumb: 'Affinity' }}>
+            <Route
+              path=":antiAffinityGroup"
+              lazy={() =>
+                import('./pages/project/affinity/AntiAffinityGroupPage.tsx').then(convert)
+              }
+            />
+          </Route>
```

Comment:
This way expresses the situation more clearly and lets you share the crumb (remember to take the handle out of `AffinityPage.tsx`)

```tsx
<Route path="affinity" handle={{ crumb: 'Affinity' }}>
  <Route
    index
    lazy={() => import('./pages/project/affinity/AffinityPage.tsx').then(convert)}
  />
  <Route
    path=":antiAffinityGroup"
    lazy={() =>
      import('./pages/project/affinity/AntiAffinityGroupPage.tsx').then(convert)
    }
  />
</Route>
```